pycparser/c_parser.py

```diff
@@ -616,6 +616,17 @@ def p_statement(self, p):
         """
         p[0] = p[1]
 
+    def p_pragmacomp_or_statement(self, p):
+        """ pragmacomp_or_statement	: pppragma_directive statement
```

Comment:
Can the pragma come after the statement?

Reply:
I can't think of a situation where a statement followed by a pragma would be a case for grouping with a compound. The pragma is the decorator for the following statement but the statement preceding a pragma wouldn't mean anything to the pragma.